docs/_install-and-build/Installing-Geth.md

```diff
@@ -28,12 +28,6 @@ You can install the develop branch using the `--devel` parameter:
 brew install ethereum --devel
 ```
 
-After installing, run `geth account new` to create an account on your node.
-
-You should now be able to run `geth` and connect to the network.
```

Comment:
I wouldn't remove this line, because it's a logical step to check if your installation was successful.